src/Microsoft.Azure.WebJobs.Host/JobHostConfiguration.cs

```diff
@@ -197,6 +197,12 @@ public string StorageConnectionString
             set { _storageAccountProvider.StorageConnectionString = value; }
         }
 
+        /// <summary>
+        /// Gets or sets a SAS connection string for a container. If this is set, <see cref="StorageConnectionString"/> and <see cref="DashboardConnectionString"/>
+        /// can be set to null and the runtime will use the container. 
+        /// </summary>
+        public string SaSContainerConnectionString { get; set; }
```

Comment:
Perhaps rename to BlobContainerSAS? If we ever support additional SAS resources (Tables, Queues) it might be good to have the names scoped to services.

Actually, if we think there is a possibility that we might move in that direction. I suggest we add a new JobHostSASConfiguration object and corresponding SASConnections property. That gives us room to expand SAS support in the future w/o creating a bunch of top level properties.

Reply:
Also, it looks like there is a hard coded precedence now - SAS before dedicated for certain system services. Should probably document that?

Reply:
I'll add JobHostSASConfiguration. I expect this config part to get superseded once we finish the DI work.